packages/flutter_test/lib/src/goldens.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:typed_data';
 
+import 'package:flutter/material.dart';
```

Comment:
This additional import is .. surprising?

Reply:
My IDE has a sense of humor. Thanks.